src/utils/exportUtils/PlainTextExport.ts

```diff
@@ -122,7 +123,11 @@ export default class PlainTextExporter extends Exporter {
             if (this.cancelled) return this.cleanUp();
             if (!haveRendererForEvent(event, false)) continue;
             const textForEvent = await this.plainTextForEvent(event);
-            content += textForEvent && `${new Date(event.getTs()).toLocaleString()} - ${textForEvent}\n`;
+            content +=
+                textForEvent &&
+                `${new Date(event.getTs()).toLocaleString("en-US", {
```

Comment:
The locale should not be fixed to US. Pass an empty array here to use whatever locale the user has configured.
```suggestion
                `${new Date(event.getTs()).toLocaleString([], {
```

Reply:
got it...thanks